packages/context-entries/lib/Keys.ts

```diff
@@ -294,3 +299,42 @@ export const KeysRdfJoin = {
    */
   lastPhysicalJoin: new ActionContextKey<string>('@comunica/bus-rdf-join:lastPhysicalJoin'),
 };
+
+export const KeysStatisticsTracker = {
+  /**
+   * Map holding all statistics being tracked, filled with keys from KeysTrackableStatistics
+   */
+  statistics: new ActionContextKey<Map<IActionContextKey<any>, any>>(
```

Comment:
I would introduce a new class/type for these statistics, as `Map<IActionContextKey<any>, any>` is not that nice, and not type-safe.

Reply:
It might be overkill, but could it also be an action context?

Reply:
Probably won't work, as that one is immutable, whereas we do want something mutable here.
But something similar type-wise could work.